test/files/neg/t8855.check

```diff
@@ -0,0 +1,8 @@
+t8855.scala:5: error: stable identifier required, but 42 found.
+    import x._
+           ^
+t8855.scala:6: warning: _root_ in root position of qualifier refers to the root package, not value _root_, which is in scope
+    val g: Int => Int = +
+               ^
```

Comment:
Wait, what is _this_?

Reply:
It sounds like a Groucho Marx line. I'll take a look, since I obviously didn't look before.

Reply:
I may add `--no-look` as an alias for `--update-check`.

Reply:
Apparently the import does import `_root_`, and parser makes the function type `_root_.scala.Function1`. Why does the import do that?

Reply:
Or `--fingers-crossed`. That should also commit, push, and request paulp's review.

Reply:
It's due to importing from error type, I guess, or maybe no type. BTW, shell history says I didn't actually see the warning; I'd compiled with doti, then changed `Function1` to fat arrow and used :crossed_fingers: option.

Reply:
"Groucho Marx line": "How a root package got in my pajamas, I'll never know."